README.md

```diff
@@ -8,6 +8,21 @@ This repository contains the **source code** of the official app registry for th
  </a>
 </p>
 
+## Requirements
+
+Before adding an app to the registry, make sure that the following requirements (required by the European Commission) are fulfilled:
+
+- The app's hosted source code repository (e.g., on GitHub) is publicly accessible.
+- The repository specifies the app's open-source license (e.g., in `license.txt`). You can find a list of approved open-source licenses [here](https://opensource.org/licenses).
+- The repository makes suitable acknowledgment to the funding by the European Commission (e.g., in `README.txt`):
+
+```text
+This project has received funding from the European Union’s Horizon 2020 research and innovation programme under grant agreement No 957189.
+```
+
+- The repository provides sufficient technical documentation on how to install and run the software. This can be achieved via a README file, a Wiki page, or a software documentation hosting platform such as [Read The Docs](https://readthedocs.org/). Please pay particular attention to this, since it is a criterion
+the evaluators will use to test the apps.
```

Comment:
Why is everything in one line except for line 24? In general, the README should follow the "one sentence per line" rule, unless this doesn't work for the list.

Reply:
Thanks for spotting this mistake.